### DISCUSSION TITLE
Units of return values of inbuilt functions

### DISCUSSION THREAD

**Discussion by AakashJ2412** [bulletphysics/bullet3#3424] [General]
What are the units of the return values of getJointStates() function, for position, velocity, and torque? Also, what is the return value of the function call calculateInverseDynamics? The code used to call these functions is:
`joint_states = bullet.getJointStates(id_robot, id_revolute_joints)`
`torque = bullet.calculateInverseDynamics(id_robot, obj_pos, obj_vel, obj_acc)`

**Comment by rohit-kumar-j** [comment]
@AakashJ2412 
SI units: Kg, Sec, Meter, Newtons. 
See #3807
